micromanager_gui/_core_widgets/_property_browser.py

```diff
@@ -0,0 +1,189 @@
+from functools import partial
+from typing import Dict, Iterator, Optional, Set, Tuple, cast
+
+from pymmcore_plus import CMMCorePlus, DeviceType
+from qtpy.QtCore import Qt
+from qtpy.QtGui import QColor
+from qtpy.QtWidgets import (
+    QCheckBox,
+    QDialog,
+    QGridLayout,
+    QGroupBox,
+    QHBoxLayout,
+    QLineEdit,
+    QPushButton,
+    QTableWidget,
+    QTableWidgetItem,
+    QVBoxLayout,
+    QWidget,
+)
+
+from .._core import get_core_singleton
+from ._property_widget import PropertyWidget
+
+
+def iter_dev_props(mmc: CMMCorePlus) -> Iterator[Tuple[str, str]]:
+    for dev in mmc.getLoadedDevices():
+        for prop in mmc.getDevicePropertyNames(dev):
+            yield dev, prop
+
```

Comment:
same comment about replace with pymmcore-plus method

Reply:
lets wait for that to be released and then do a cleanup here